doc/api/cli.md

```diff
@@ -1370,8 +1370,15 @@ added: v12.0.0
 -->
 
 This configures Node.js to interpret `--eval` or `STDIN` input as CommonJS or
-as an ES module. Valid values are `"commonjs"` or `"module"`. The default is
-`"commonjs"`.
+as an ES module. Valid values are `"commonjs"`, `"module"`, `"module-typescript"` and `"commonjs-typescript"`.
+The `"-typescript"` values are available only in combination with the flag `--experimental-strip-types`.
```

Comment:
Just confirming, I assume that once we unflag strip-types then all that happens is that this sentence disappears? We don’t create some other new flag that needs to be passed to permit these new values?

Reply:
Exactly no need for additional flags